client/client.go

```diff
@@ -219,6 +226,17 @@ func WithHTTPClient(client *http.Client) func(*Client) error {
 	}
 }
 
+// WithHijackDialer overrides the client default hijack dialer.
+// The typical usecase of WithHijackDialer is to allow implementing connection helpers.
+// The hijack dialer typically corresponds to *http.Client.Transport.(*http.Tranport).DialContext .
+// See WithHTTPClient.
```

Comment:
Instead of corresponds to, maybe "See http.Transport.DialContext for more information" ?